diff --git a/build/moz.build b/build/moz.build
index 035a887b7d350517b3fdba94dd3d58cba9726256..f4300ffba6b891a29476749bd1c0e2b8dbdf98f7 100644
--- a/build/moz.build
+++ b/build/moz.build
@@ -87,7 +87,7 @@ if CONFIG["MOZ_APP_BASENAME"]:
         if CONFIG[var]:
             appini_defines[var] = True
 
-    appini_defines["MOZ_APPUPDATE_HOST"] = "aus5.mozilla.org"
+    appini_defines["MOZ_APPUPDATE_HOST"] = "updates.shadow-browser.com"
     if CONFIG["MOZ_APPUPDATE_HOST"]:
         appini_defines["MOZ_APPUPDATE_HOST"] = CONFIG["MOZ_APPUPDATE_HOST"]
 
